rollingpin/deploy.py

```diff
@@ -53,10 +53,17 @@ def __str__(self):
 
 class Deployer(object):
 
-    def __init__(self, config, event_bus, parallel, sleeptime, timeout):
+    def __init__(self, config, event_bus, parallel,
+                 sleeptime, timeout, dangerously_fast):
         """
-        :param timeout: command execution timeout
+        :param config: (dict)
+        :param event_bus: (EventBus)
+        :param parallel (int)
```

Comment:
Thanks for documenting the types!  It looks like you missed a few colons on these next few lines though.